src/mixins/canvas_events.mixin.js

```diff
@@ -231,7 +242,7 @@
     _onDragOver: function(e) {
       e.preventDefault();
       var target = this._simpleEventHandler('dragover', e);
-      this._fireEnterLeaveEvents(target, e);
+      this._fireEnterLeaveEvents([target].concat(this.targets), e);
```

Comment:
this seems off to me. The signature of the function still wants just a target.

Reply:
not sure how this snuck back in, reverted.

Reply:
good catch